### DISCUSSION TITLE
Will this library be merged with mysqljs/mysql in the future?

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [sidorares/node-mysql2#1515] [Q&A] [answered]
Hi. i was reading mysql2's documentation and i saw something about working with mysqljs/mysql developers and so on ... . So my question is: will this library be merged with mysqljs/mysql in the future?

**Comment by sidorares** [comment, marked as answer]
It's unlikely that it's going to be "merged" - quite a lot if internals are different between 2 libraries. However I'm more than happy to see common parts ( constants, helper libraries ) to be factored out and reused (examples of what already happened - https://www.npmjs.com/package/sqlstring and https://www.npmjs.com/package/named-placeholders ). Also external test harness for the public api would be also great - that would allow to remove big part of unit tests and focus only on testing extra  functionality

Having said that, if you really do want to see "merged" future, splitting into smaller parts is probably the best way to get there.

**Reply by MadadiSaeed** [reply]
Thanks for response.
